docs/source/pages/get_started.md

```diff
@@ -0,0 +1,183 @@
+# Get Started
+
+This is a brief introduction to getting started with DataShuttle, containing a subset of the full documentation.
+For more detail, please see the full documentation.
+
+This documentation gives examples both using the API (in the python console) or using the command line interface (in system terminal).
+
+## Installation
+
+DataShuttle is hosted on  [PyPI](https://pypi.org/project/datashuttle/) and can be installed with pip.
+
+`pip install datashuttle`
+
+Datashuttle required Rclone for data transfers. The easiest way to install Rclone is using [Miniconda](https://docs.conda.io/en/main/miniconda.html):
+
+```
+conda install -c conda-forge rclone
+```
+
+See [the Rclone website](https://rclone.org/install/) for alternative installation methods.
+
+## Initial Setup - Set Configurations
+
+To get started, import DataShuttle and initialise the class with the project name. If using the command
+line this step is not necessary.
+
+```
+from datashuttle.datashuttle import DataShuttle
+
+project = DataShuttle("my_project")
+```
+
+DataShuttle helps to manage and transfer a project with many "local" machines all connected to a central "remote" machine.
```

Comment:
The concept of a "project" isn't explained. I would explain it a bit, and link to the swc-blueprint docs for the user to read first.